DQM/Integration/python/clients/beamhltfake_dqm_sourceclient-live_cfg.py

```diff
@@ -1,39 +1,21 @@
 from __future__ import print_function
+import time
```

Comment:
is this import actually needed?

Reply:
in fact it is needed only in one of the two. I will removed it from the other